pkgs/leak_tracker/lib/src/leak_tracking/primitives/_test_helper_detector.dart

```diff
@@ -27,13 +29,7 @@ final _startFrame = RegExp(
   r')',
 );
 
-/// Returns whether the leak reported by [objectCreationTrace]
-/// was created by a test helper.
-///
-/// Frames, that match [exceptions] will be ignored.
-///
-/// See details on what means to be created by a test helper
-/// in doc for `LeakTesting.createdByTestHelpers`.
+@visibleForTesting
```

Comment:
nit: why remove dart-doc? it still helps to explain what this method does even though it is now private but visibileForTesting